docs/src/concrete-playback.md

```diff
@@ -0,0 +1,59 @@
+# Concretely playing back proof harnesses
```

Comment:
I expect the title here to match the section name: "Debugging harness"

Reply:
done